experimental/BasicAsyncHttpServer/BasicAsyncHttpServer.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net451</TargetFramework>
+    <!-- TODO remove rid when https://github.com/dotnet/sdk/issues/396 is resolved -->
+    <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
```

Comment:
(affects multiple files). As it only targets net451, you can remove the condition.